macro/o2sim.C

```diff
@@ -25,13 +25,27 @@
 #include <SimSetup/SimSetup.h>
 #include <Steer/O2RunSim.h>
 #include <DetectorsBase/MaterialManager.h>
+#include <CCDB/BasicCCDBManager.h>
 #include <unistd.h>
 #include <sstream>
 #endif
 
 FairRunSim* o2sim_init(bool asservice)
 {
   auto& confref = o2::conf::SimConfig::Instance();
+  // initialize CCDB service
+  auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
+  ccdbmgr.setURL(confref.getConfigData().mCCDBUrl);
+  ccdbmgr.setTimestamp(confref.getConfigData().mTimestamp);
+  // try to query test object to verify connection
+  auto testobj = ccdbmgr.get<TObject>("Test");
+  if (testobj != nullptr) {
```

Comment:
@sawenzel actually, why do report error on non-0 pointer and not vice versa?

I see something weird here: ``ccdbmgr.get<TObject>("Test")`` always returns 0
````
root [0] auto& mgr = o2::ccdb::BasicCCDBManager::instance()
(o2::ccdb::BasicCCDBManager &) @0x7f15626c8018
root [1] auto& confref = o2::conf::SimConfig::Instance();
root [2] auto& ccdbmgr = o2::ccdb::BasicCCDBManager::instance();
root [3] ccdbmgr.setURL(confref.getConfigData().mCCDBUrl);
root [4] ccdbmgr.setTimestamp(confref.getConfigData().mTimestamp);
root [5] std::cout << ccdbmgr.getTimestamp() << "\n";
1567183368791
root [6] std::cout << ccdbmgr.get<TObject>("Test") << "\n";
0
root [7] .!date
Fr Aug 30 18:42:55 CEST 2019
````
But then, if I look on http://ccdb-test.cern.ch:8080/browse/Test?report=true
I see an object created at approximate time of my query and covering the time stamp I've queried.

Reply:
weird. Will take a look.

Reply:
I think I understand:
a) there is a typo in the check
b) The API assumes that the blob is stored as TMemFile whereas the "Test" folder probably still uses TMessage.

I should really be querying something else here. At some moment the TMemFile will be consistently used so these inconsistencies will disappear.